frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -1403,6 +1403,10 @@ export const dashboardLogic = kea<dashboardLogicType>([
                 .sort((a, b) => (a.order ?? 0) - (b.order ?? 0))
                 .map((t) => t.insight)
                 .filter((i): i is QueryBasedInsightModel => !!i)
+                // only refresh stale insights
+                .filter(
+                    (i) => manualDashboardRefresh || !i.cache_target_age || dayjs(i.cache_target_age).isBefore(dayjs())
```

Comment:
this would mean that insights with time grouped by 'min' (which I think have cache lasting less than an hour) will not be refreshed if they were last refreshed earlier in the day?

I think this is fine, as no matter the insight grouping if it's refreshed any point today